target/linux/mvebu/cortexa9_neon/base-files/etc/uci-defaults/03_wireless

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+
+[ ! -e /etc/config/wireless ] && exit 0
+
+. /lib/functions.sh
+. /lib/functions/system.sh
+
+board=$(board_name)
+
+case "$board" in
+linksys,caiman|linksys,cobra|linksys,shelby|linksys,venom)
```

Comment:
I'd prefer the vertical stack here (like in the majority of the other files)
option1|\
option2|\
option3)